content/blog/2022/02/2022-02-10-last-plugin-version-not-installable.adoc

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: "Why can't I install the latest plugin on my Jenkins LTS?"
+tags:
+- jenkins
+- plugin
+- upgrade
+- developer
+authors:
+- jmMeessen
+description: >
+  This blogpost explains why the latest version of a plugin can't be installed on Jenkins in some circumstances.
+opengraph:
+  image: /images/post-images/2022-02-10-last-plugin-version-not-installable/summary_picture.png
+---
+
+I regularly go, as all Jenkins admin do, to my system's Plugin Management page to see what plugins can be upgraded.
+
+Although I am running the latest link:/download/lts/[LTS version], **I can't update some key plugins** (example the Junit plugin).
+This is even more intriguing as some are part of the recommended plugins (default plugin choice).
+No "install" tick box and the plugin is labeled "unavailable".
+There is a note saying "_This version of the plugin exists but it is not being offered for installation, so the latest bug fixes or features are not available to you. This is typically the case when plugin requirements, e.g. a recent version of Jenkins, are not satisfied. If you are using the latest version of Jenkins offered to you, newer plugin releases may not be available to your release line yet. See the plugin documentation for information about its requirements._".
+
+image:/images/post-images/2022-02-10-last-plugin-version-not-installable/plugin-not-installable.png[pluginView]
+
+The message indicates to Jenkins Administrators that a newer plugin version is available 
+but that the installed Jenkins core doesn't provide the necessary prerequisite.
+
+**Why is this happening now?**
+Did I miss something?
+
+As part of the continuous improvement of Jenkins, some critical and high impact changes are introduced.
+This is typically the case of GUI changes (icons, fonts, layout) as is currently happening.
+It is also the case for new core features that can be consumed by plugins.
+
+Such changes are typically first introduced in the cutting edge Jenkins "weekly release".
```

Comment:
Unusual to put it in quotes, I don't think we do that anywhere else. It's just the name.